src/var.jl

```diff
@@ -0,0 +1,39 @@
+export similarvariable, @similarvariable
+
+"""
+    similarvariable(p::AbstractPolynomialLike, variable::Type{Val{V}})
+
+Creates a new variable `V` based upon the the given source polynomial.
+
+    similarvariable(p::AbstractPolynomialLike, v::Symbol)
+
+Creates a new variable based upon the given source polynomial and the given symbol `v`. Note
+that this can lead to type instabilities.
+
+### Examples
+
+Calling `similarvariable(typedpoly, Val{:x})` on a polynomial created with `TypedPolynomials`
+results in `TypedPolynomials.Variable{:x}`.
+"""
+function similarvariable end
```

Comment:
No, I mean to add `similarvariable` it in the `docs/src/apireference.md` between line 10 and line 11.